_sources/scripts/run-maintenance-scripts.sh

```diff
@@ -361,25 +361,16 @@ run_autoupdate () {
 }
 
 run_import () {
-    # Import dumps if any
+    # Import PagePort dumps if any
     if mountpoint -q -- "$MW_IMPORT_VOLUME"; then
-        echo "Found $MW_IMPORT_VOLUME, checking for XML dump presence.."
-        local FILES_IMPORTED=0
-        for filename in $MW_IMPORT_VOLUME/*.xml; do
-            echo "Found $filename file, importing.."
-            php maintenance/importDump.php \
-                --username-prefix="" \
-                --memory-limit=max \
-                --report 1 \
-                "$filename"
-            FILES_IMPORTED=$((FILES_IMPORTED + 1))
-        done
-        if [ "$FILES_IMPORTED" -gt "0" ]; then
-            echo "Running post-import maintenance scripts.."
-            php maintenance/rebuildrecentchanges.php
-            php maintenance/initSiteStats.php
+        echo "Found $MW_IMPORT_VOLUME, running PagePort import.."
+        XML_TEST=($(find $MW_IMPORT_VOLUME -maxdepth 1 -name "*.xml"))
+        if [ ${#XML_TEST[@]} -gt 0 ]; then
+        	echo "WARNING! The directory contains XML files, did you forget to migrate the dump to PagePort?"
+        	return
         fi
-        echo "Imported $FILES_IMPORTED files!"
+        php extensions/PagePort/maintenance/importPages.php --source "$MW_IMPORT_VOLUME"
```

Comment:
So I tried this, and it worked - two small issues
```
2024-08-31 10:37:40 web-1            | Done!Imported completed!
```
should be fixed by WikiTeq/PagePort#9

And also, on my test wiki the edit was created to the flow talk page manager, which is apparently who the context user was - can we please pass a user via the `--user` parameter? Suggest using the `Maintenance script` user from MediaWiki core